src/Silverback.Integration.Kafka.Testing/Messaging/Broker/Kafka/Mocks/InMemoryTopic.cs

```diff
@@ -237,6 +237,11 @@ private void ScheduleRebalance(string groupId)
 
         private void Rebalance(string groupId)
         {
+            if (groupId == "not-set")
```

Comment:
Why? Is this necessary?
The actual broker will not be aware of our convention at threat `"not-set"` as any other group.id.

Reply:
Code removed: https://github.com/BEagle1984/silverback/pull/172/commits/da1f43cd3b9f655e5b2c4b0759c8cf8bbf06707b